themes/default/content/what-is/resolve-list-buckets-signature-does-not-match.md

```diff
@@ -0,0 +1,110 @@
+---
+title: An error occurred (SignatureDoesNotMatch) when calling the ListBuckets operation
+allow_long_title: true
+meta_desc: |
+     Use Pulumi ESC and dynamic credentials to run commands like aws ListBuckets in a more secure and seamless way.
+type: what-is
+page_title: An error occurred (SignatureDoesNotMatch) when calling the ListBuckets operation
+---
+
+The error message "An error occurred (SignatureDoesNotMatch) when calling the ListBuckets operation" in AWS (Amazon Web Services) indicates that there's a mismatch between the request signature calculated by AWS and the signature you provided. AWS relies on two key components for authentication: the AWS Access Key Id and the AWS Secret Access Key. The Secret Access Key is used for securely signing requests to AWS services. When you encounter this error, it typically signifies that the provided AWS Secret Access Key is incorrect or has been used with an inappropriate signing method.
+
+[Pulumi ESC (Environments, Secrets, and Configurations)](/docs/pulumi-cloud/esc/) offers a solution to the challenges of managing credentials and can make errors like this a thing of the past. By enabling the [management of dynamic credentials from AWS using OIDC](/blog/esc-env-run-aws/), Pulumi ESC simplifies and secures your AWS CLI operations. This approach eliminates the need for manually providing credentials and is a more secure solution than the use of long-term credentials, which can often present a security risk. Pulumi ESC also enables you to focus on your tasks without the interruption of credential-related errors, providing a more efficient flow for your AWS operations and tasks.
```

Comment:
This could use the additional context tied the fact you no longer have to use manually input in Secret and Access keys

Reply:
That is already in the "what is esc run" section

Reply:
"This approach eliminates the need for manually providing credentials" is already there. I think "credentials" as an umbrella term is a more efficient approach than having to spell everything out individually.